server/engine/src/utils/scale_osim.py

```diff
@@ -0,0 +1,76 @@
+import nimblephysics as nimble
+from typing import Dict, List, Tuple
+import tempfile
+import shutil
+import numpy as np
+import subprocess
+
+def scale_osim(unscaled_generic_osim_text: str,
```

Comment:
Consider `scale_opensim_model`.

Reply:
Also consider renaming the test files accordingly.

Reply:
Done